client/src/pages/CreateModal.js

```diff
@@ -0,0 +1,158 @@
+import { UserContext } from "../contexts/UserContext";
+
+const useStyles = makeStyles((theme) => ({
+  backdrop: {
+    width: "100vw",
+    height: "100vh",
+    position: "fixed",
+    top: "0",
+    left: "0",
+    zIndex: "2",
+    background: "rgba(0,0,0,0.6)",
+  },
+  modal: {
+    display: "flex",
+    background: "#ffffff",
+    padding: "1rem",
+    width: "400px",
+    height: "320px",
+    borderRadius: "8px",
+    boxShadow: "0px 0px 10px 5px rgba(0,0,0,0.1)",
+  },
+  modalMain: {
+    height: "80%",
+  },
+  title: {
+    fontWeight: "800",
+    marginTop: "0.5rem",
+  },
+  createButton: {
+    padding: "1rem 3rem",
+    borderRadius: "8px",
+    background: "#759CFC",
+    textTransform: "capitalize",
+    "&:hover": {
+      background: "#759CFC",
+    },
+  },
+  input: {
+    width: "300px",
+  },
+  close: {
+    width: "100%",
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "flex-end",
+    color: "#ccc",
+  },
+}));
+function CreateModal(props) {
+  const classes = useStyles();
+  const { setShowModal, type } = props;
+  const { user } = useContext(UserContext);
+
+  const [title, setTitle] = useState("");
+
+  const handleCreate = (data, type) => {
+    switch (type) {
+      case "board": {
+        const cleanedData = {
+          userId: data._id,
+          title: title,
+        };
+        createBoard(cleanedData);
+        break;
+      }
+      case "column": {
+        const cleanedData = {
+          boardId: data.boards[0], //TODO: make dynamic
+          title: title,
+        };
+        createColumn(cleanedData);
+        break;
+      }
+      case "card": {
+        const cleanedData = {
+          columnId: "602f2b85e297d244f8f90d1c" || props.columnId, //PENDING: switch dummyData to DB
+          title: title,
+        };
+        createCard(cleanedData);
+        break;
+      }
+      default:
+        return;
+    }
+  };
+  const handleClick = () => {
+    handleCreate(user, type);
+    setShowModal(false);
+  };
+
+  return (
+    <Grid
+      container
+      direction="column"
+      justify="center"
+      alignItems="center"
+      className={classes.backdrop}>
+      <Paper className={classes.modal}>
+        <Grid
+          container
+          direction="column"
+          justify="flex-start"
+          alignItems="center">
+          <Grid item className={classes.close}>
+            <IconButton onClick={() => setShowModal(false)}>
+              <CloseIcon />
+            </IconButton>
+          </Grid>
+          <Grid
+            container
+            direction="column"
+            justify="space-between"
+            alignItems="center"
+            className={classes.modalMain}>
+            <Grid item>
+              <Typography variant="h4" className={classes.title}>
+                Create a new {type}
+              </Typography>
+            </Grid>
+            <Grid item>
+              <TextField
+                defaultValue="Add Title"
```

Comment:
```suggestion
                placeholder="Add Title"
```
This way the user doesn't need to delete it